requirements.txt

```diff
@@ -3,6 +3,6 @@ netCDF4
 matplotlib
 vtk
 vtk_visualizer
-opencv-python
+opencv-python==4.2.0.34
```

Comment:
This should be a separate commit message with good explanation.

Reply:
This issue was found and fixed in a new PR, approved by Trym.